core/includes/menu.inc

```diff
@@ -539,8 +539,9 @@ function menu_execute_active_handler($path = NULL, $deliver = TRUE, $route_handl
   // Offline pages are not cached in the database but have a Cache-Control
   // header to allow reverse proxies to serve offline hits instead of Backdrop.
   elseif ($site_status == MENU_SITE_OFFLINE) {
-    $page_cache_enabled = (bool) config_get('system.core', 'page_cache_maximum_age');
-    $maintenance_max_age = config_get('system.core', 'maintenance_page_maximum_age');
+    $site_config = config('system.core');
```

Comment:
You could even move that up to the beginning of function menu_execute_active_handler(). - In line 524 there's already a call to `config_get('system.core', 'menu_route_handler');`

Reply:
Nice catch! Thanks for the suggestion. Committed.